models/public/yolo-v1-tiny-tf/keras_to_tensorflow.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/keras_to_tensorflow.py b/keras_to_tensorflow.py
+index c66cbbd..6290ed5 100644
+--- a/keras_to_tensorflow.py
++++ b/keras_to_tensorflow.py
+@@ -16,7 +16,7 @@ from pathlib import Path
+ from absl import app
+ from absl import flags
+ from absl import logging
+-import keras
++from tensorflow import keras
```

Comment:
Why is this patch necessary? Don't you have Keras in the dependency list?

Reply:
Because of this error https://stackoverflow.com/questions/58878421/unexpected-keyword-argument-ragged-in-keras

Reply:
Keras package is also required

Reply:
How bizarre. OK.